docs/docs-content/clusters/public-cloud/azure/azure-cloud.md

```diff
@@ -21,121 +21,98 @@ Palette supports deploying and managing Kubernetes clusters in an Azure account.
 
 To register an Azure cloud account in the Palette console
 
-1. Log in to [Palette](https://console.spectrocloud.com).
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
 
+2. From the left **Main Menu**, select **Tenant Settings**. 
 
-2.  Navigate to the **Project Overview** drop-down and switch to the **Tenant Admin**. 
+3. Select **Cloud Accounts** in the Tenant Settings menu 
 
+4. Locate **Azure**, and click **+ Add Azure Account**.
 
-3.  Select **Tenant Settings** from the left **Main Menu**. 
+5. Fill out the following information, and click **Confirm** to complete the registration.
 
-
-4. From the Tenant Settings go to **Cloud Accounts** and click on **+ Add Azure Account**.
-
-
-5. The Azure cloud account wizard requires the following information:
-
-|   **Basic Information** |Description|
+|   **Basic Information** |**Description**|
 |-------------------------|-----------|
-|Account Name| A custom account name|
-|Client ID| Unique client Id from Azure console|
-|Tenant ID| Unique tenant Id from Azure console|
-|[Client Secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#create-an-azure-active-directory-application)| Azure secret for authentication|
-|Tenant Name| An optional tenant name|
-|[Disable Properties](/clusters/public-cloud/azure/azure-cloud#disableproperties)| To disable the import of Azure networking details.|
-|Toggle **Connect Private Cloud Gateway**| An option to select the [Self-Hosted PCG](gateways.md) already created from the drop-down menu to link it to the cloud account. |
-
-:::info
-
-  For existing cloud accounts go to **Edit** and toggle the **Connect Private Cloud Gateway** option to select the created Gateway from the drop-down menu.
-:::
-
-
-6. Click on the **Confirm** button to complete the wizard.
+|**Account Name**| A custom account name.|
+|**Tenant ID**| Unique tenant ID from Azure Management Portal.|
+|**Client ID**| Unique client ID from Azure Management Portal.|
+|**Client Secret**| Azure secret for authentication. Refer to Microsoft's reference guide for creating a [Client Secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#create-an-azure-active-directory-application) |
```

Comment:
There is a validate button right next to the credentials. We need to mention this.

![CleanShot 2023-11-21 at 19 09 10@2x](https://github.com/spectrocloud/librarium/assets/29551334/1b3b1df8-3db3-4550-b9d4-924555b8df93)